src/main/java/seedu/jarvis/ui/MasteryCheckListPanel.java

```diff
@@ -8,7 +8,7 @@
 import javafx.scene.control.ListView;
 import javafx.scene.layout.Region;
 import seedu.jarvis.commons.core.LogsCenter;
-import seedu.jarvis.model.consultation.MasteryCheck;
+import seedu.jarvis.model.masterycheck.MasteryCheck;
```

Comment:
Since MasteryCheck is using a separate tab from Consultations, Making Mastery check no longer extend Consultation makes sense now.

Reply:
Ah, it still extends Consultation! I just moved it to a separate package since it was getting a bit crowded in there. :)